src/screens/Button.js

```diff
@@ -683,6 +683,15 @@ const ButtonPage = () => (
           </PropertyValue>
         </Property>
 
+        <Property name="button.active.default.icon">
```

Comment:
Do we need to add these or is this covered by the docs for `button.active.default`?

Reply:
good question I'm open to either @jcfilben ? any opinions?

Reply:
Good point, I think we could leave `button.active.default.icon` out. Maybe on the `button.active.default` it would be good to clarify what props can be passed in there.